src/pages/PostsList.vue

```diff
@@ -0,0 +1,112 @@
+<template>
+  <q-page class="page-post-list flex justify-center">
+    <div class="page-post-list__table-list full-width q-px-xl">
+      <div class="bg-info page-post-header row justify-between">
+        <div class="text-h1 q-my-lg">Blog</div>
+        <q-btn rounded color="primary" label="Novo Post" to="createAuthor" class="self-end"/>
```

Comment:
esse "to="createAuthor"" esta correto? se for name precisa passar dentro do objeto